build.sbt

```diff
@@ -1,3 +1,5 @@
 libraryDependencies += "org.scalatest" %% "scalatest" % "2.1.5" % "test"
 
-libraryDependencies += "org.apache.httpcomponents" % "httpclient" % "4.3.4"
\ No newline at end of file
+libraryDependencies += "org.apache.httpcomponents" % "httpclient" % "4.3.4"
+
+libraryDependencies += "com.typesafe.play" %% "play-json" % "2.4.0-M3"
```

Comment:
@phamann stable versions are available from the Typesafe repo e.g. http://repo.typesafe.com/typesafe/releases/com/typesafe/play/play-json_2.10/
You can add it as an additional resolver to your `plugins.sbt` e.g.
```
// The Typesafe repository 
resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/"
```